mobile/src/libs/components/input-search/input-search.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { type Dispatch, type SetStateAction } from 'react';
+
+import { TextInput, View } from '#libs/components/components';
+import { AppColor } from '#libs/enums/enums';
+import { debouncedFuncion as debouncedSetSearchQuery } from '#libs/helpers/helpers';
+import { useAppForm, useFormController } from '#libs/hooks/hooks';
+
+import { styles } from './styles';
+
+type Properties = {
+  placeholder: string;
+  setSearchQuery: Dispatch<SetStateAction<string>>;
+};
+
+const InputSearch: React.FC<Properties> = ({ placeholder, setSearchQuery }) => {
+  const { control } = useAppForm({
+    defaultValues: { search: '' },
+  });
+  const { field } = useFormController({ name: 'search', control });
+  const { value, onChange } = field;
```

Comment:
move default values to input-search/libs/constants as it's done for sign up

Reply:
Done